zh_CN/data-integration/schema-registry.md

```diff
@@ -20,20 +28,26 @@ Schema Registry 既可以解码，也可以编码。Schema Registry 为 Avro 和
 
 编码调用示例：
 
-```c
-schema_encode(SchemaName, Data) -> RawData
+```erlang
+schema_encode(SchemaName, Map) -> Bytes
 ```
 
 解码调用示例：
 
-```c
-schema_decode(SchemaName, RawData) -> Data
+```erlang
+schema_decode(SchemaName, Bytes) -> Map
 ```
 
 当对 JSON 格式的 MQTT 消息进行编码时，在用 schema 编码之前，您也需要用 `json_decode` 先对其进行解码，使它变为规则引擎内部数据格式 (Map)，示例如下：
 
 ```erlang
-schema_encode(SchemaName, json_decode(Data))
+schema_encode(SchemaName, json_decode(Map)) -> Bytes
+```
+
+在编码前或解码后检查 JSON 数据是否可以根据 JSON schema 进行验证时，使用以下编解码验证示例：
+
+```erlang
+schema_check(SchemaName, Map | Bytes) -> Boolean
```

Comment:
This function needs to be added to the rule-sql-builtin-functions.md section.

Reply:
done